exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -0,0 +1,131 @@
+# Default Registry and MF1 Compatibility
+
+Status: **Proposed**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@aphillips</dd>
+		<dt>First proposed</dt>
+		<dd>2023-12-15</dd>
+	</dl>
+</details>
+
+## Objective
+
+_What is this proposal trying to achieve?_
+
+This section contains the list of functions (both _selectors_ and _formatters_)
```

Comment:
Looks like there is quite a bit of duplication between this and what we already have in the `registry.xml`

What I thing is hight priority to address:
 - selectors, including decision on names: plural (cardinal & ordinal, with offset), select (or `string`?), maybe gender (maybe not now)
 - do we really want all the aliases?

I am afraid that we are reopening things that are already decided (formatters), and nobody complained about.

Reply:
This is, in fact, copied from `registry.xml` on purpose (thank you for making the original registry file). The only differences here should be the things proposed by @eemeli in the number selector PR. I wanted to have a way to discuss what goes into 2.0 without the clutter and to illustrate where we are vs. MF1.